examples/main.cpp

```diff
@@ -195,6 +195,8 @@ int run(const char *device_name,
     uq_policy = AMSUQPolicy::DeltaUQ_Max;
   else if (strcmp(uq_policy_opt, "deltauq-mean") == 0)
     uq_policy = AMSUQPolicy::DeltaUQ_Mean;
+  else if (strcmp(uq_policy_opt, "random") == 0)
+    uq_policy = AMSUQPolicy::RandomUQ;
```

Comment:
I think we have a bug in the uq_policy_opt flag.It is set on a default value of "faiss-mean". What happens if the user does not specify the uq_policy_opt flag and does not provide an FAISS index. Should we just default to random?

Reply:
You're right. How about we remove the default and we error check in the UQ constructor that there is always a path when specifying FAISS policies?

Reply:
Sounds good.